code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -42,10 +42,6 @@
 		return			// Suppress emote, you're still cooling off.
 
 	switch(act)
-		if("me")									//OKAY SO RANT TIME, THIS FUCKING HAS TO BE HERE OR A SHITLOAD OF THINGS BREAK

```

Comment:
This rant is here for a reason, humans do not call parent in their /emote, because of so much unique code. They need this here, or you get things like drugs spamming the user with `Unknown emote, me`